### ISSUE TITLE
DeepScanner: a Robotic System for Automated 2D Object Dataset Collection with Annotations. (arXiv:2108.02555v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#9636]
DeepScanner: a Robotic System for Automated 2D Object Dataset Collection with Annotations. (arXiv:2108.02555v1 [cs.RO])<br>
https://ift.tt/37kcbAW<br>
<p>In the proposed study, we describe the possibility of automated dataset collection using an articulated robot. The proposed technology reduces the number of pixel errors on a polygonal dataset and the time spent on manual labeling of 2D objects. The paper describes a novel automatic dataset collection and annotation system, and compares the results of automated and manual dataset labeling. Our approach increases the speed of data labeling 240-fold, and improves the accuracy compared to manual labeling 13-fold. We also present a comparison of metrics for training a neural network on a manually annotated and an automatically collected dataset.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
